src/Twig/Extension.php

```diff
@@ -814,6 +814,7 @@ public function getTests()
     {
         return [
             new TwigTest('numeric', [$this, 'isNumeric']),
+            new TwigTest('instanceof', [$this, 'isInstanceOf']),
```

Comment:
Ik vroeg me af (ook al eens in een ander geval) hoe het precies loopt als twee Twig Extensions dezelfde functie of test definiëren. Stel dat ik zelf in mijn project al een `instanceof` test heb en ik update naar deze versie van de Framework bundle, wat gebeurt er dan? Een error? De ene overschrijft de andere, en welke?

Reply:
Dat zal ik even uitzoeken :female_detective: Ik kom er op terug :+1:

Reply:
Het lijkt erop dat de versie in deze bundel voorrang krijgt. Als je in een project dus exact dezelfde naam voor een test hebt, wordt deze overschreven.

Reply:
Je krijgt overigens geen error.

Reply:
Is het niet handiger om de parameter te typehinten?

Dan kan je een error terug sturen als de parameter value is geen object.

Reply:
:+1: 

Ik gok dan dat het de eerste bundle is (op de volgorde in config/bundles.php) die een functie registreert. Dus functies worden niet overschreven, denk ik hieruit op te maken.